src/main/java/com/github/splendor_mobile_game/game/model/Chat.java

```diff
@@ -0,0 +1,63 @@
+package com.github.splendor_mobile_game.game.model;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+public class Chat {
+    private ArrayList<ChatMessage> chatMessages = new ArrayList<>();
+
+    public ArrayList<ChatMessage> getChatMessages() {
+        Collections.sort(chatMessages);
```

Comment:
As far as i know - using ArrayLists, automatically adds item in some order. So if you add a message to chatMessages, then you don't need to sort it, because it should be stored in the same order as you have inserted it